apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -1 +1,31 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
 import './lib/changeIdButton.js';
+import { saveButtonEnabling } from './lib/saveButtonEnabling.js';
+
+onDocumentReady(() => {
+  const qidfield = document.querySelector('input[name="qid"]') as HTMLInputElement;
```

Comment:
Nit (note that you'll have to change this elsewhere below too):

```suggestion
  const qidField = document.querySelector('input[name="qid"]') as HTMLInputElement;
```